config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-1.10.yaml

```diff
@@ -1,91 +1,133 @@
+      - name: AWS_K8S_TESTER_EKS_ENABLE_NODE_SSH
+        value: "true"
+      # 'true' to enable S3 Access Logs and AWS ALB Access Logs
+      # use it for debug, dump cluster log already handles log artifacts
+      - name: AWS_K8S_TESTER_EKS_LOG_ACCESS
+        value: "false"
+      # 'true' to upload 'aws-k8s-tester' logs to S3 buckets, in addition to log dumper
+      # use it for debug, dump cluster log already handles log artifacts
+      - name: AWS_K8S_TESTER_EKS_UPLOAD_TESTER_LOGS
+        value: "false"
+      # 'true' to upload worker node logs to S3, in addition to log dumper
+      # use it for debug, dump cluster log already handles worker node logs
+      - name: AWS_K8S_TESTER_EKS_UPLOAD_WORKER_NODE_LOGS
+        value: "false"
+      # 'true' to enable debug level logs
+      - name: AWS_K8S_TESTER_EKS_LOG_DEBUG
+        value: "false"
+      # 'true' to create AWS ALB
+      - name: AWS_K8S_TESTER_EKS_ALB_ENABLE
+        value: "false"
+      # AWS test account credential mounted path, required for AWS API call
+      - name: AWS_SHARED_CREDENTIALS_FILE
+        value: /etc/eks-aws-credentials/eks-aws-credentials
+    volumeMounts:
+      - mountPath: /etc/eks-aws-credentials
+        name: eks-aws-credentials
+        readOnly: true
+    volumes:
+      - name: eks-aws-credentials
+        secret:
+          secretName: eks-aws-credentials
+
+  # configuration for EKS Kubernetes e2e tests with 1.10
+  - labels:
+      preset-kubernetes-e2e-aws-eks-1-10: "true"
+    env:
+      # URL to download 'kubectl', required for 'kubectl' calls to EKS
+      - name: AWS_K8S_TESTER_EKS_KUBECTL_DOWNLOAD_URL
+        value: https://amazon-eks.s3-us-west-2.amazonaws.com/1.10.3/2018-07-26/bin/linux/amd64/kubectl
+      # URL to download 'aws-iam-authenticator', required for 'kubectl' calls to EKS
+      - name: AWS_K8S_TESTER_EKS_AWS_IAM_AUTHENTICATOR_DOWNLOAD_URL
+        value: https://amazon-eks.s3-us-west-2.amazonaws.com/1.10.3/2018-07-26/bin/linux/amd64/aws-iam-authenticator
+      # configure EKS Kubernetes version
+      - name: AWS_K8S_TESTER_EKS_KUBERNETES_VERSION
+        value: "1.10"
+      # Amazon EKS-optimized AMI (non-GPU, https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
+      - name: AWS_K8S_TESTER_EKS_WORKER_NODE_AMI
+        value: ami-0f54a2f7d2e9c88b3
+      # worker node EC2 instance type
+      - name: AWS_K8S_TESTER_EKS_WORKER_NODE_INSTANCE_TYPE
+        value: m3.xlarge
+      # worker node auto-scaling group minimum number of nodes
+      - name: AWS_K8S_TESTER_EKS_WORKER_NODE_ASG_MIN
+        value: "1"
+      # worker node auto-scaling group maximum number of nodes
+      - name: AWS_K8S_TESTER_EKS_WORKER_NODE_ASG_MAX
+        value: "1"
+
+presubmits:
+  # Run Kubernetes 1.10 branch e2e tests with EKS prod build 1.10
+  # run conformance tests
+  kubernetes/kubernetes:
+    - name: pull-kubernetes-e2e-1-10-aws-eks-1-10-prod-conformance
+      branch:
+      - master
```

Comment:
I'm confused here, do you want to run against 1.10 branch or master branch? The job indicates 1.10?

Reply:
Yeah for now, we want to run this against master branch (given that 1.10 would be deprecated soon, and we are working to support later versions). Once we get this to work, we will rename it.